build.sh

```diff
@@ -31,8 +31,6 @@ function print_help {
     echo "        Platform(s) to build, defaults to desktop."
     echo "        Building for Android or iOS will automatically generate / download"
     echo "        the toolchains if needed and perform a partial desktop build."
-    echo "    -r"
```

Comment:
Did we remove the feature? I thought we had issues in CI (macOS I believe with too many open files).

Reply:
If we ever supported this feature, it was before our GitHub history began.  Yeah I vaguely remember that issue.  Hm.

Reply:
Is it gone from the rest of the script? We definitely used it in the Kokoro build scripts.

Reply:
Correct.